hack/util.sh

```diff
@@ -228,7 +228,8 @@ function ensure_iptables_or_die {
 # tryuntil loops, retrying an action until it succeeds or times out after 90 seconds.
 function tryuntil {
 	timeout=$(($(date +%s) + 90))
-	echo "++ Retrying until success or timeout: ${@}"
+	echo "++ Retrying until success or timeout: $*"
+	# shellcheck disable=SC2078
 	while [ 1 ]; do
```

Comment:
Would using a more shell-idiomatic `while :` or `while true` suppress the warning?